charactersheet/charactersheet/templates/common/party_manager.tmpl.html

```diff
@@ -105,20 +105,27 @@ <h4 class="modal-title" id="importModalLabel">
           <div class="row row-padded">
             <label for="roomName"
                    class="col-sm-3 control-label">Party Link</label>
-            <div class="col-sm-8">
-              <input type="text" disabled
+            <div class="col-sm-7">
+              <input type="text" readonly id="roomLink"
                      class="form-control"
                      data-bind="value: roomLink">
             </div>
+            <div class="col-sm-1">
```

Comment:
Can you make this an add-on to the right of the input?

http://getbootstrap.com/components/#input-groups-buttons